spec/webview-spec.ts

```diff
@@ -1955,6 +1955,35 @@ describe('<webview> tag', function () {
       }, [fixtures]);
     });
 
+    describe('<webview>.getNavigationEntryForIndex()', () => {
```

Comment:
We'll want to add similar tests to `api`-web-contents-spec`!

Reply:
Good call! Missed this test file. Will add!